src/app/custom-keyboard/_components/BoardOption.tsx

```diff
@@ -0,0 +1,82 @@
+'use client';
+
+import classNames from 'classnames/bind';
+import { useContext } from 'react';
+import { KeyboardDataContext } from '@/context/customKeyboardContext';
+import { HexColorPicker } from 'react-colorful';
+import { CustomKeyboardTypeTypes, CustomKeyboardTextureTypes } from '@/types/CustomKeyboardTypes';
```

Comment:
```suggestion
import type { CustomKeyboardTypeTypes, CustomKeyboardTextureTypes } from '@/types/CustomKeyboardTypes';
```
type을 import하실때에는 이렇게 적어주는게 좋다고 생각합니다
개인 취향이에요!

Reply:
적용 하신다면 모든 컴포넌트에 신경써주시는게 좋을거같습니다

Reply:
@bokeeeey 그때 말씀하신게 이거군요 오홍,, 신기하네요,.